open-craft/opencraft
# [SE-3646] Add management command to drop and recreate failed instance DBs.

This pull request makes it possible to drop and recreate the database of an instance that failed provisioning.

**JIRA tickets**: https://tasks.opencraft.com/browse/SE-3646

**Merge deadline**: "None" if there's no rush, "ASAP" if it's critical, or provide a specific date if there is one.

**Testing instructions**:

1. Create a new instance and interrupt the provisioning process after the database is created but before provisioning is complete.
2. Run the command as described in the new documentation.
3. Check the instance logs to verify that the information about the admin and their reasoning is present.
3. Provision a new app server and verify that it is able to complete using the recreated database.
4. After provisioning completes successfully, attempt to run the command on the instance again. It should fail.

**Reviewers**
